### ISSUE TITLE
Error Retrieving  values of FLOAT types columns 

### ISSUE THREAD

**Issue by akhtar-a** [codenotary/immudb4j#64]
<img width="1202" alt="Screenshot 2024-05-30 at 10 18 54 AM" src="https://github.com/codenotary/immudb4j/assets/66638688/5694939e-047f-4978-b5a0-affae16f5b21">

<img width="1360" alt="Screenshot 2024-05-30 at 10 15 50 AM" src="https://github.com/codenotary/immudb4j/assets/66638688/a913f070-48ef-4688-90d7-6760f556ab11">

<img width="999" alt="Screenshot 2024-05-30 at 10 14 32 AM" src="https://github.com/codenotary/immudb4j/assets/66638688/efa98315-0dac-4fc4-91af-e1e38c0da19b">

response.getLong() , .getString , getBytes nothing works, I am not able to retrieve the value, It also Shows unknown in the debugger

**Comment by akhtar-a** [comment]
@ostafen Any Idea how do we actually get FLOAT values ?

**Comment by akhtar-a** [comment]
<img width="839" alt="Screenshot 2024-05-30 at 4 08 34 PM" src="https://github.com/codenotary/immudb4j/assets/66638688/5614fb95-f94d-4520-9ecb-e870334150cd">

**Comment by ostafen** [comment]
Thank you for opening the issue, I will check

**Comment by akhtar-a** [comment]
@ostafen  Any Updates on this ?

**Comment by ostafen** [comment]
@akhtar-a, we are working on it and it will be released shortly.
As a temporary solution, you can CAST FLOAT to VARCHAR when returning the column and then parse it from string directly in the code.

**Comment by akhtar-a** [comment]
@ostafen  Yes we have done that only for now, I wanted to increment and avoid multiple reads which is not possible as of now with VARCHAR type of values

UPDATE table 
value = value + float(SOME FLOAT NUMNER)
where id=1
So would be great to have this. 🥇

**Comment by ostafen** [comment]
You don't have to store the column as VARCHAR.
You cast store it as FLOAT and cast it to VARCHAR while specifying target expressions on the select statement.
